shardingsphere-governance/shardingsphere-governance-repository/shardingsphere-governance-repository-provider/shardingsphere-governance-repository-etcd/src/main/java/org/apache/shardingsphere/governance/repository/etcd/EtcdRepository.java

```diff
@@ -109,11 +109,12 @@ public void watch(final String key, final DataChangedEventListener dataChangedEv
             for (WatchEvent each : response.getEvents()) {
                 ChangedType changedType = getEventChangedType(each);
                 if (ChangedType.IGNORED != changedType) {
-                    dataChangedEventListener.onChange(new DataChangedEvent(each.getKeyValue().getKey().toString(Charsets.UTF_8), each.getKeyValue().getValue().toString(Charsets.UTF_8), changedType));
+                    dataChangedEventListener.onChange(new DataChangedEvent(each.getKeyValue().getKey().toString(StandardCharsets.UTF_8)
+                            , each.getKeyValue().getValue().toString(StandardCharsets.UTF_8), changedType));
```

Comment:
`,` must be on the first line.

Reply:
@menghaoranss first line means the same indent?

Reply:
```
dataChangedEventListener.onChange(new DataChangedEvent(each.getKeyValue().getKey().toString(StandardCharsets.UTF_8),
                            each.getKeyValue().getValue().toString(StandardCharsets.UTF_8), changedType));
```

Reply:
@TanveshT Hi, any other questions?

Reply:
Nothing else @menghaoranss. Will make the that commit soon